thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/DashboardResource.java

```diff
@@ -558,4 +565,29 @@ public String getData(@QueryParam("type") String type) throws Exception {
     }
     return response;
   }
+
+  @POST
+  @Path("/severity")
+  public Response computeSeverity(@NotNull @QueryParam("collection") String collectionName,
```

Comment:
suggest moving this to another Resource, may be DetectionJobResource or even Admin is also fine.